Source/player.h

```diff
@@ -527,6 +527,17 @@ struct Player {
 		return blkper;
 	}
 
+	/**
+	 * @brief Return reciprocal of the factor for calculating damage reduction due to Mana Shield.
+	 *
+	 * Valid only for players with Mana Shield spell level greater than zero.
+	 */
+	int GetManaShieldDamageReduction()
+	{
+		constexpr int8_t Max = 7;
+		return 24 - std::min(_pSplLvl[SPL_MANASHIELD], Max) * 3;

```

Comment:
MSVC is having none of this:
![image](https://user-images.githubusercontent.com/204594/150279302-87b6d0d0-d8a9-452b-a21c-a3f82093af7a.png)

Reply:
Probably missing `#include <algorithm>`

Reply:
I guess it would make sense to move the definition to player.cpp, then.